client/my-sites/sidebar/site-menu.jsx

```diff
@@ -90,7 +90,6 @@ class SiteMenu extends PureComponent {
 				name: 'post',
 				label: translate( 'Posts' ),
 				capability: 'edit_posts',
-				config: 'manage/posts',
```

Comment:
We should be able to remove the config logic from https://github.com/Automattic/wp-calypso/pull/53455/files#diff-29e15167224ca54ba114cb703ae0632c1101679db5374851b00a0f2bc025cbb5R153 now, right?

Reply:
Yes we can! But I need to verify that more carefully and wanted to focus this PR strictly on removing flags that are obviously always true.

Reply:
Follow-up sounds good to me obviously 👍